### PR TITLE
Feature/8 thymeleaf

### PR DESCRIPTION
**Pull request by starnowski**

### CODE REVIEW

File: posmulten-hibernate5-spring-boot-thymeleaf/.gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+**/.idea
+
+# idea subdirectories
+**/.idea/*
+
+#maven
+target/*
+
+**/target/*
```
Comment (mkyc) [inline posmulten-hibernate5-spring-boot-thymeleaf/.gitignore:29]: missing `\n`

File: posmulten-hibernate5-spring-boot-thymeleaf/pom.xml [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok-mapstruct-binding</artifactId>
+                            <version>0.2.0</version>
+                        </path>
+                    </annotationProcessorPaths>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (mkyc) [inline posmulten-hibernate5-spring-boot-thymeleaf/pom.xml:260]: I'm not going to comment in every file but there is quite clear [definition of line](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_205) 😉

File: posmulten-hibernate5-spring-boot-thymeleaf/src/main/java/com/github/starnowski/posmulten/demos/posmultenhibernate5springbootthymeleaf/filters/TenantFilter.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.github.starnowski.posmulten.demos.posmultenhibernate5springbootthymeleaf.filters;
+
+public class TenantFilter extends AbstractTenantFilter {
```
Comment (mkyc) [inline posmulten-hibernate5-spring-boot-thymeleaf/src/main/java/com/github/starnowski/posmulten/demos/posmultenhibernate5springbootthymeleaf/filters/TenantFilter.java:3]: shouldn't it be `RequiredTenantFilter`?

File: posmulten-hibernate5-spring-boot-thymeleaf/src/main/webapp/WEB-INF/templates/add-users.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        </div>
+        <div class="form-group">
+            <label class="col-lg-3 control-label">Password</label>
+            <div class="col-lg-9">
+                <input type="password" class="form-control" placeholder="Password (as required)" th:field="*{password}"/>
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label class="col-lg-3 control-label">Password</label>
```
Comment (mkyc) [inline posmulten-hibernate5-spring-boot-thymeleaf/src/main/webapp/WEB-INF/templates/add-users.html:32]: Password?

File: posmulten-hibernate5-spring-boot-thymeleaf/pom.xml (hunk 2 above)
Comment (starnowski) [reply posmulten-hibernate5-spring-boot-thymeleaf/pom.xml:260]: Fixed in https://github.com/starnowski/posmulten-demos/pull/11

File: posmulten-hibernate5-spring-boot-thymeleaf/src/main/java/com/github/starnowski/posmulten/demos/posmultenhibernate5springbootthymeleaf/filters/TenantFilter.java (hunk 3 above)
Comment (starnowski) [reply posmulten-hibernate5-spring-boot-thymeleaf/src/main/java/com/github/starnowski/posmulten/demos/posmultenhibernate5springbootthymeleaf/filters/TenantFilter.java:3]: Fixed in https://github.com/starnowski/posmulten-demos/pull/11

File: posmulten-hibernate5-spring-boot-thymeleaf/src/main/webapp/WEB-INF/templates/add-users.html (hunk 4 above)
Comment (starnowski) [reply posmulten-hibernate5-spring-boot-thymeleaf/src/main/webapp/WEB-INF/templates/add-users.html:32]: Fixed in https://github.com/starnowski/posmulten-demos/pull/11

File: posmulten-hibernate5-spring-boot-thymeleaf/.gitignore (hunk 1 above)
Comment (starnowski) [reply posmulten-hibernate5-spring-boot-thymeleaf/.gitignore:29]: Fixed in https://github.com/starnowski/posmulten-demos/pull/11

**Pull request merged**